llvm/lib/YkIR/YkIRWriter.cpp

```diff
@@ -465,6 +458,19 @@ class YkIRWriter {
     for (unsigned OI = 0; OI < I->arg_size(); OI++) {
       serialiseOperand(I, VLMap, I->getOperand(OI));
     }
+    if (!I->getCalledFunction()->isDeclaration()) {
+      // The next instruction will be the stackmap entry
+      CallInst *SMI = dyn_cast<CallInst>(I->getNextNonDebugInstruction());
+      assert(SMI);
+      assert(SMI->getCalledFunction()->isIntrinsic());
+      assert(SMI->getIntrinsicID() == Intrinsic::experimental_stackmap);
+      // has_safepoint = 1:
+      OutStreamer.emitInt8(1);
+      serialiseDeoptSafepoint(SMI, VLMap);
```

Comment:
This definitely should be called `serialiseStackmap` because the argument SMI is a stackmap instruction.

Reply:
I still believe this should be called `serialiseStackmapCall` as that's the concrete name of the value we pass in and are serialising into what we from then on call "safepoint".

Reply:
Yeah, agreed. Fixed here (6ba0599b239791a7b52190ad48b745717373f195)